sources/packages/backend/apps/api/src/services/education-program-offering/custom-validators/has-valid-offering-period-for-funded-days.ts

```diff
@@ -5,78 +5,75 @@ import {
   ValidationOptions,
   ValidationArguments,
 } from "class-validator";
-import { OFFERING_STUDY_PERIOD_MAX_DAYS } from "../../../utilities";
 import { StudyBreak } from "../education-program-offering-validation.models";
 import { OfferingCalculationValidationBaseConstraint } from "./offering-calculation-validation-base-constraint";
 
 /**
  * For an offering that contains study breaks, execute the calculation
- * of the funded study period to validate if it matches the mix and max
- * allowed study period amount of days.
+ * of the funded study period to validate if it matches the minimum
+ * allowed study period amount of weeks.
  */
 @ValidatorConstraint()
-class HasValidOfferingPeriodForFundedDaysConstraint
+class HasValidOfferingPeriodForFundedWeeksConstraint
   extends OfferingCalculationValidationBaseConstraint
   implements ValidatorConstraintInterface
 {
   validate(studyBreaks: StudyBreak[], args: ValidationArguments): boolean {
-    const offeringMinDaysAllowedValue = this.getOfferingMinAllowedDays(args);
+    const offeringMinWeeksAllowedValue = this.getOfferingMinAllowedWeeks(args);
     const calculatedStudyBreaksAndWeeks = this.getCalculatedStudyBreaks(
       studyBreaks,
       args,
     );
     return (
-      calculatedStudyBreaksAndWeeks.fundedStudyPeriodDays >=
-        offeringMinDaysAllowedValue &&
-      calculatedStudyBreaksAndWeeks.fundedStudyPeriodDays <=
-        OFFERING_STUDY_PERIOD_MAX_DAYS
+      calculatedStudyBreaksAndWeeks.totalFundedWeeks >=
+      offeringMinWeeksAllowedValue
     );
   }
 
   defaultMessage(args: ValidationArguments) {
-    const offeringMinDaysAllowedValue = this.getOfferingMinAllowedDays(args);
-    return `The funded study amount of days is ineligible for StudentAid BC funding. Your dates must be between ${offeringMinDaysAllowedValue} to ${OFFERING_STUDY_PERIOD_MAX_DAYS} days.`;
+    const offeringMinWeeksAllowedValue = this.getOfferingMinAllowedWeeks(args);
+    return `The funded study amount of days is ineligible for StudentAid BC funding. Your offering must be at least ${offeringMinWeeksAllowedValue} weeks of study or longer to be eligible.`;
```

Comment:
This message is displayed in UI only during bulk offering upload. Please double check with business on the content of this message. As the AC did not state what must be the message for min weeks validation on bulk offerings upload.